holopy/core/io/vis.py

```diff
@@ -72,6 +72,48 @@ def show(o, scaling='auto', vert_axis='x', horiz_axis='y',
     else:
         raise VisualizationNotImplemented(o)
 
+def save(o, names, scaling='auto', vert_axis='x', horiz_axis='y',
```

Comment:
Can we change the function name to a more descriptive name? Perhaps `save_reconstruction` or `save_plotted_hologram`?

Reply:
As the method is able to save holograms and reconstruction, I relabeled it to `save_plot`. Is this better?

Reply:
Additional comment: Shouldn't `show` then be renamed, too? As they process the same type of data?